cluster/saltbase/salt/top.sls

```diff
@@ -58,6 +61,9 @@ base:
     - kube-controller-manager
     - kube-scheduler
     - supervisor
+{% if grains['cloud'] is defined and not grains.cloud in ['aws', 'gce', 'vagrant', 'vsphere'] %}
```

Comment:
Sorry, what I meant was, why isn't this
`{% if grains['cloud'] is defined and grains['cloud'] == 'azure-legacy' %}`
?